hawk_test.py

```diff
@@ -75,7 +75,7 @@ def main():
     ssh = HawkTestSSH(args.host, args.secret)
 
     # Get version from /etc/os-release
-    test_version = ssh.ssh.exec_command("grep ^VERSION= /etc/os-release")[1].read().decode().strip().split("=")[1].strip('"')
+    test_version = ssh.ssh.exec_command("grep VERSION= /etc/os-release")[1].read().decode().strip().split("=")[1].strip('"')
```

Comment:
this one come from https://github.com/ClusterLabs/hawk/pull/267/files#diff-a961d2e4b10784f32e4a3e57c70da34df75dc60a3fefc4482bb863d36cb6504eR78

Reply:
yes, thank you for helping to comment it :+1: